src/typing_extensions.py

```diff
@@ -1543,7 +1379,7 @@ def get_origin(tp):
         """
         if isinstance(tp, _AnnotatedAlias):
             return Annotated
-        if isinstance(tp, (typing._GenericAlias, _typing_GenericAlias, _BaseGenericAlias,
+        if isinstance(tp, (_typing_GenericAlias, _BaseGenericAlias,
```

Comment:
is this change correct? As I mentioned above, `typing._GenericAlias` is a different class to `_typing_GenericAlias` -- the latter actually appears to be `types.GenericAlias`

Reply:
Seems like we need more tests here too

Reply:
The existing `isinstance()` check had `typing._GenericAlias`, which was useless as `typing._GenericAlias` is a subclass of `_BaseGenericAlias` and this one is already in the tuple.

Thus I removed `typing._GenericAlias`, matching [the current implementation](https://github.com/python/cpython/blob/3.9/Lib/typing.py#L1538).

Reply:
We do have one already:

https://github.com/python/typing_extensions/blob/4525e9dbbd177b4ef8a84f55ff5fe127582a071d/src/test_typing_extensions.py#L1783

Reply:
Thanks!